core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Authenticator.java

```diff
@@ -80,11 +87,22 @@ protected Optional<Identity> createIdentity(String token)
         return Optional.of(builder.build());
     }
 
+    private Optional<TokenPair> deserializeToken(String token)
+    {
+        try {
+            return Optional.of(tokenPairSerializer.deserialize(token));
+        }
+        catch (RuntimeException ex) {
```

Comment:
nit: Can we use `exception` ?

Reply:
It would be too broad. Also there are no checked exceptions, as compiler would complain.

Reply:
I meant the variable name. nvm